### PR TITLE
finished drone list

### PR DESCRIPTION
**Pull request by Matyasai**
…heir status

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './styles/droneList.css'
+
+const drones = [
```
Comment (tomoiagahoratiu) [inline src/main.tsx]: I would create a models directory, where i would create:
```
type Drone = {
    id: number;
    status: DroneStatus;
};
```
And the DroneStatus
`type DroneStatus = "parked" | "delivering" etc...;`

Comment (tomoiagahoratiu) [reply src/main.tsx]: then i would assign the Drone type to the drones constant

File: src/main.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+function Main() {
+    let list = [];
+    for(let i = 0; i < drones.length; i++) {
+        if(drones[i].status === "parked") {
+            list[i] = <li className="green">
+                {drones[i].id} - {drones[i].status};
+            </li>
+        }
+        else if(drones[i].status === "delivering") {
```
Comment (tomoiagahoratiu) [inline src/main.tsx]: I would modify the following code:
create a switch block function and based on the drones status return the correct css class name.
then I would create the list like this 
```
const dronesList = drones.map((drone, index) => (
    <li key={index} className={getClassName(drone.status)}>
        {drone.id} - {drone.status}
    </li>
```

File: src/styles/droneList.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+ol {
```
Comment (tomoiagahoratiu) [inline src/styles/droneList.css]: I would also play a little bit with the css and maybe make the list of drones to look more like the design on figma

File: src/main.tsx (hunk 1 above)
Comment (Matyasai) [reply src/main.tsx]: Done

File: src/main.tsx (hunk 2 above)
Comment (Matyasai) [reply src/main.tsx]: Done

File: src/styles/droneList.css (hunk 3 above)
Comment (Matyasai) [reply src/styles/droneList.css]: Will start from tomorrow

**Comment by tomoiagahoratiu** [comment]
Can you please rebase this pr with main?

**Comment by Matyasai** [comment]
> Can you please rebase this pr with main?

rebased with main @tomoiagahoratiu @tomoiagahoratiu

**Comment by tomoiagahoratiu** [comment]
there are still some issues with the PR, did you add the logic that was already added by Andrei with your commits? Because when looking at your changes you have a lot of changes related to other of your colleagues work.

**Pull request closed without merging**